modules/model/item/components/overcast.js

```diff
@@ -19,6 +19,7 @@ export class OvercastItemModel extends BaseItemModel {
                additional  : new fields.StringField(),
                characteristic  : new fields.StringField(),
                bonus : new fields.BooleanField(),
+               cost : new fields.StringField()
```

Comment:
I don't see this referenced anywhere else other than on the Sheet. Is this utilized anywhere? 

If so, what is the point of having this as well as "other" in config?

Reply:
The idea is to be able to create a formula, that can be calculated per overcast 'level' its in WoM cast test used - 
![image](https://github.com/user-attachments/assets/6dd4496a-d900-4aa1-a691-75c765c04630)

I didnt add it to regular CastTest, as there overcast rules are super straightforward.